### PR TITLE
Issue #388: Don't output ancestry to beanstalkd

### PR DESCRIPTION
**Pull request by akotlar**
* Write ancestry results to disk instead of sending the entire AncestryResult object through beanstalkd.

As currently architected, there is no way for Ancestry to scale beyond about 3000 samples, because 1) beanstalkd is not designed to send objects of 20+ MB (it can, but communication slows, clogs up queue), 2) mongodb can only store 16MB of data for the entire job, of which ancestry is just a part, 3) users almost certainly want to be able to download results.

In this PR we write to the user specified output directory, and return the name of the file written. We write a json file, uncompressed.

### CODE REVIEW

File: python/python/bystro/ancestry/dummy_ancestry_response.py [hunk 1]
```diff
@@ -1,150 +0,0 @@
-"""Provide dummy ancestry response for integration testing."""
```
Comment (akotlar) [inline python/python/bystro/ancestry/dummy_ancestry_response.py:1]: This file was unused and redundant with the test_ancestry_listener.py examples on how to build an AncestryJobCompleteMessage

**Comment by cristinaetrv** [comment]
General question - is this how we'll want to handle outputs for all analysis modules going forward since they'll all be similar sizes or larger?

**Review by cristinaetrv — APPROVED** [review]
Lgtm!

**Review by dlin30 — COMMENTED** [review]
LGTM

**Pull request merged**